apps/charterafrica/.env.template

```diff
@@ -20,6 +20,8 @@ NEXT_PUBLIC_APP_URL
 NEXT_PUBLIC_DEFAULT_LOCALE
 NEXT_PUBLIC_IMAGE_DOMAINS
 NEXT_PUBLIC_IMAGE_UNOPTIMIZED
+NEXT_PUBLIC_APP_AUTH_EMAIL
+NEXT_PUBLIC_APP_AUTH_PASSWORD
```

Comment:
Do we need any auth? I believe all collections allow anyone to read e.g. for Pages: https://github.com/CodeForAfrica/ui/blob/1462db9951eb2fd1dc164dcc83753a212b21facc/apps/charterafrica/src/payload/collections/Pages.js#L19-L21

Reply:
On the API, I get this error without any authentication

<img width="721" alt="image" src="https://user-images.githubusercontent.com/39160236/218114428-b418c2fb-1def-490a-9572-cd6349b59adf.png">

Reply:
I ran the PR locally without providing my email/password and it worked. May be it was that hard-coded json file. Let me try again @koechkevin

Reply:
Just tried again and I can pull the content without auth info @koechkevin

Reply:
Could it be something I am missing?  I keep getting this.
<img width="1187" alt="image" src="https://user-images.githubusercontent.com/39160236/218444120-3d7a0125-a555-4e58-af0e-f4ac41bf4cd2.png">

Reply:
similar t when I try to access https://charterafrica.prod.codeforafrica.org/api/globals/footer

Unless I register and login.

Reply:
Could be... Why are you connecting to prod though? Have you tried dev or local/Docker payload @koechkevin ?

Reply:
Does it work on local/Docker @koechkevin ?

Reply:
Not connecting to prod.

Just that the responses are similar.

Reply:
Ah!

Just found out the problem